src/coreclr/src/jit/lower.h

```diff
@@ -283,17 +283,22 @@ class Lowering final : public Phase
 #endif // defined(TARGET_XARCH)
 
     // Per tree node member functions
+    void LowerStoreIndirCommon(GenTreeIndir* ind);
+    void LowerIndir(GenTreeIndir* ind);
     void LowerStoreIndir(GenTreeIndir* node);
     GenTree* LowerAdd(GenTreeOp* node);
     bool LowerUnsignedDivOrMod(GenTreeOp* divMod);
     GenTree* LowerConstIntDivOrMod(GenTree* node);
     GenTree* LowerSignedDivOrMod(GenTree* node);
     void LowerBlockStore(GenTreeBlk* blkNode);
+    void LowerBlockStoreCommon(GenTreeBlk* blkNode);
```

Comment:
Unless I am missing something, these could be merged; I don't understand the reason for a separate method.

Reply:
Do you mean `LowerBlockStore` and `LowerBlockStoreCommon`? The first is defined separately for armarch and xarch.

Reply:
Oh I see - I missed that; thanks!